src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -9,7 +9,11 @@ string: (IDENT | STRING_LITERAL);
 integer: INTEGER_LITERAL;
 charset_name: (IDENT | STRING_LITERAL | QUOTED_IDENT);
 
-SQL_COMMENT: '/*' (.)*? '*/' -> skip;
+SQL_UPGRADE_COMMENT: '/*!' [0-9]* -> skip;
+SQL_UPGRADE_ENDCOMMENT: '*/' -> skip;
+
+SQL_COMMENT: '/*' ~'!' (.)*? '*/' -> skip;
```

Comment:
the behavior of antlr's lexer is to match the longest token first, so we have to make sure that `SQL_COMMENT` dodges `/*!`, otherwise the greedy little thing that antlr is will gobble it.